Tests/CucumberMessages.CompatibilityTests/CucumberMessages.Tests.csproj

```
@@ -0,0 +1,262 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Samples\Resources\ambiguous\Ambiguous.cs" />
+    <Compile Remove="Samples\Resources\attachments\attachments.cs" />
+    <Compile Remove="Samples\Resources\background\background.cs" />
+    <Compile Remove="Samples\Resources\cdata\cdata.cs" />
+    <Compile Remove="Samples\Resources\data-tables\data_tables.cs" />
+    <Compile Remove="Samples\Resources\empty\empty.cs" />
+    <Compile Remove="Samples\Resources\examples-tables-attachment\examples_tables_attachment.cs" />
+    <Compile Remove="Samples\Resources\examples-tables\examples_tables.cs" />
+    <Compile Remove="Samples\Resources\external-data\external_data.cs" />
+    <Compile Remove="Samples\Resources\ExternalBindingAssemblies\SampleExternalBindingClass.cs" />
+    <Compile Remove="Samples\Resources\ExternalBindingAssemblies\SampleInternalBindingClass.cs" />
+    <Compile Remove="Samples\Resources\hooks-attachment\hooks_attachment.cs" />
+    <Compile Remove="Samples\Resources\hooks-conditional\hooks_conditional.cs" />
+    <Compile Remove="Samples\Resources\hooks-named\hooks_named.cs" />
+    <Compile Remove="Samples\Resources\hooks\hooks.cs" />
+    <Compile Remove="Samples\Resources\minimal\minimal.cs" />
+    <Compile Remove="Samples\Resources\parameter-types\parameter_types.cs" />
+    <Compile Remove="Samples\Resources\pending\pending.cs" />
+    <Compile Remove="Samples\Resources\retry-NUnitRetry\retry_NUnitRetry.cs" />
+    <Compile Remove="Samples\Resources\retry-xRetry\retry_xRetry.cs" />
+    <Compile Remove="Samples\Resources\rules\rules.cs" />
+    <Compile Remove="Samples\Resources\skipped\skipped.cs" />
+    <Compile Remove="Samples\Resources\stack-traces\stack_traces.cs" />
+    <Compile Remove="Samples\Resources\test-run-hooks\test_run_hooks.cs" />
+    <Compile Remove="Samples\Resources\undefined\undefined.cs" />
+    <Compile Remove="Samples\Resources\unknown-parameter-type\unknown_parameter_type.cs" />
```

Could this be simplified?

```suggestion
    <Compile Remove="Samples\Resources\**\*cs" />
```

---

I've adopted this suggestion.